cypress/e2e/undo-redo/block-configs-should-update-flow-state/numeric-block-configs.cy.js

```diff
@@ -1,120 +0,0 @@
-describe('Completely configure numeric block and apply undo/redo to changes', () => {
-    beforeEach(() => {
-    cy.createFlow({
-    label: 'Undo/Redo numeric block configs ',
-    })
-})
-    
-    
-    it('should configure and test the block', () => {
-        const blockConfigs = {
-            label: 'your age',
-            name: {
-                beforeEdit: 'your_age',
-                afterEditPostFix: '_updated',
-            },
-            resources: {
-                ivr: ' 02_flowers_for_albert.mp3',
-                sms: 'Kindly enter your age',
-                ussd: 'Kindly enter your age',
-            },
-            values: 
-            {
-                minimum: 12,
```

Comment:
no need to retest common configs @pascaleli , we'd just focus on those 03 specific configs for this block